tests/reprsimil/test_gbrsa.py

```diff
@@ -0,0 +1,331 @@
+#  Copyright 2016 Mingbo Cai, Princeton Neuroscience Instititute,
```

Comment:
The tests in these modules are running too long. This is why the tests are not passing. I know it is difficult, but please shorten the test to less than 1 minute in total for `reprsimil`.